diff --git a/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php b/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php
index 98228fc..34c7cdc 100644
--- a/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php
+++ b/lib/Doctrine/DBAL/Platforms/MySqlPlatform.php
@@ -530,5 +530,9 @@ SQL
             $tableOptions[] = sprintf('ROW_FORMAT = %s', $options['row_format']);
         }

+        if (!isset($options['encrypted']) || $options['encrypted']) {
+            $tableOptions[] = 'ENCRYPTED = YES';
+        }
+
         return implode(' ', $tableOptions);
     }
